### PR TITLE
Pending and completed sections

### LINKED ISSUE #108: Replace margin-left animation with translation
The CSS animations are struggling to keep to a smooth 60fps on both the target iPad device and my development machine. Looking at [this article](https://www.html5rocks.com/en/tutorials/speed/high-performance-animations/), it looks like we should be animating `translateX` instead of `margin-left`. I can see in the code that this is implemented but commented out, did it not work?

### PR DESCRIPTION
**Pull request by Boxface**
### Description

Updated the progress bar to be smoother and pushing updated sections for Pending and Completed.

### Issues fixed

Fixes #108 

### Checklist

- [X] `npm test` returns no warnings or errors.

### CODE REVIEW

File: src/components/CompletedTransactionsList/CompletedTransactionsList.story.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#root .CompletedTransactionsList .no-results {
```
Comment (ShaunBulbrook) [inline src/components/CompletedTransactionsList/CompletedTransactionsList.story.css]: Why is `#root` being used? - Can we just stick with the component selector to restrict the scope, or is this not sufficient?

File: src/components/TransactionsList/TransactionsList.story.js [hunk 2]
```diff
@@ -22,16 +22,34 @@ storiesOf('TransactionsList', module)
...
+					hash: '0.14801239520291642',
+					symbol: 'ABC',
+				},
 				{
 					from: '0.740338835632784',
 					to: '0.21455781911656757',
 					amount: '2.30',
 					timestamp: 1519121364,
-					trust: 2,
+					trust: 100,
```
Comment (ShaunBulbrook) [inline src/components/TransactionsList/TransactionsList.story.js]: @dan1elhughes - Can we get some guidance on the trust level. 
@Boxface - there might need to be some maths to be added to get these to return a percentage.

Comment (dan1elhughes) [reply src/components/TransactionsList/TransactionsList.story.js]: These are percentages, the container will convert the confirmations+trust target into a 0-100 range

Comment (ShaunBulbrook) [reply src/components/TransactionsList/TransactionsList.story.js]: ^ My bad then!

Comment (dan1elhughes) [reply src/components/TransactionsList/TransactionsList.story.js]: Down the line I'll add a custom proptype for it - in another PR though

File: src/components/CompletedTransactionsList/CompletedTransactionsList.story.css (hunk 1 above)
Comment (Boxface) [reply src/components/CompletedTransactionsList/CompletedTransactionsList.story.css]: That styling was added before I touched it. I agree that there shouldn't be any need for it, especially if the styling is restricted to the storybook CSS.

Will remove.

Comment (Boxface) [reply src/components/CompletedTransactionsList/CompletedTransactionsList.story.css]: Done

**Pull request merged**